OWNERS_ALIASES

```diff
@@ -71,11 +71,10 @@ aliases:
     - dklyle
     - rjmorse
   sig-platform-dev-leads:
-    - jzelinskie
```

Comment:
These duplicates were put there by the generator. If you run `make generate` again, does it put them back or leave them out? I don't want to drift from whatever the generator currently enforces, although we should also fix the generator to not add duplicates.